scripts/install.js

```diff
@@ -109,18 +109,68 @@ function checkAndDownloadBinary() {
       return;
     }
 
-    download(sass.getBinaryUrl(), sass.getBinaryPath(), function(err) {
-      if (err) {
-        console.error(err);
-        return;
-      }
+    var tmpPath = getTempPath(sass.getBinaryName());
+    if (!(sass.hasBinary(tmpPath))) { // download and install
```

Comment:
The name of the binary file in the tmp folder based on `sass.getBinaryName()`. E.g. `win32-x64-47_binding.node`. If you use another version of node-sass, the name will change too.